src/main/java/cloud/eppo/logging/Assignment.java

```diff
@@ -79,11 +55,11 @@ public String getSubject() {
     return subject;
   }
 
-  public String getTimestamp() {
+  public Date getTimestamp() {
```

Comment:
Moving toward the pattern of providing native Date/Time objects for the loggers, and then they can log to their warehouse however they want (vs. us being opinionated by providing an ISO string)